### PR TITLE
feat(gswd-articles): add cli app first ideas

### PR DESCRIPTION
**Pull request by asantos00**
@jonioliveira

Available at: /getting-started-with-deno/build-a-cli-with-deno

https://deploy-preview-23--alexandrempsantos.netlify.app/getting-started-with-deno/build-a-cli-with-deno/

### CODE REVIEW

File: content/blog/getting-started-with-deno/build-a-cli-with-deno.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+It is the outcome of Node's creator (Ryan Dahl) experience of using Node.js after almost a decade of leaving its core team.
+
+Ryan originally created Node.js back in 2009. Back then, Node.js brought JavaScript to the server side. Among other things, Node.js was a paradigm shift in the way many languages handled common tasks, as it deeply leveraged asynchronousity, something familiar among JavaScript users.
+
+It suddently enabled many JavaScript developers to start building server-side applications. From there, it started being used for HTTP servers, but its usecases never stopped growing, helping make JavaScript today's most popular language.
+
+**But...**
+
+However, as all the great creations, Node.js comes with its flaws. And for Ryan, its creator, they were a little itchy, especially when he used Node.js again to write simple scripts, after a few years not using it. Ryan felt like he loved the productivity and prototypability of JavaScript, but some parts of Node.js were now getting in the way, as he explained in this [talk])(https://www.youtube.com/watch?v=M3BM9TB-8yA&ab_channel=JSConf).
```
Comment (jonioliveira) [inline content/blog/getting-started-with-deno/build-a-cli-with-deno.md:18]: typo here

[talk])(https://www.youtube.com/watch?v=M3BM9TB-8yA&ab_channel=JSConf)

```suggestion
However, as all the great creations, Node.js comes with its flaws. And for Ryan, its creator, they were a little itchy, especially when he used Node.js again to write simple scripts, after a few years not using it. Ryan felt like he loved the productivity and prototypability of JavaScript, but some parts of Node.js were now getting in the way, as he explained in this [talk](https://www.youtube.com/watch?v=M3BM9TB-8yA&ab_channel=JSConf).
```

**Comment by asantos00** [comment]
TODO: update the article with the newer version of the code that contains the units

File: content/blog/deno/series-introduction.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: Exploring the deno.land - A blog post series 🦕
+description: "Introduction to a blog post series where we use Deno for multiple contexts, from building CLIs to web scrapping tools, among others."
+date: "2021-03-23"
+published: true
+---
+
+This is the introduction for a blog post series where we'll use [Deno](https://deno.land) to build different applications going from CLIs to web scrapping tools, among others.
+
+If you're not familiar with Deno, here's the [TLDR](/deno/tldr). If you at least have heard of Deno and you are here to know more about it, let's not waste you more time.
```
Comment (fsschmitt) [inline content/blog/deno/series-introduction.md]:
```suggestion
If you're not familiar with Deno, here's the [TLDR](/deno/tldr). If you have at least heard about Deno and are here to know more about it, let's not waste any more of your time.
```

File: content/blog/deno/series-introduction.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- Static site generation in Deno, with aleph.js
+- ...
+
+*Most of these posts are written in collaboration with friends working in the industry. I believe this will make such posts more interesting -- as these are people with completely different backgrounds and experiences.*
+
+We'll use these example applications to explore Deno and how it can be applied to common tasks. By the end of it, you will have a better understanding of this new JavaScript runtime and how it can be used.
+
+## Let's start!
+
+From now on it is up to you.
```
Comment (fsschmitt) [inline content/blog/deno/series-introduction.md]:
```suggestion
## Let's get hands-on

Feel free to download the code from the repo <insert-link> to make sure you not only follow through the blog post, but also try out these implementations first hand.
```

File: content/blog/deno/series-introduction.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Let's start!
+
+From now on it is up to you. 
+
+The above posts do not require to be read in any specific order, but if you want a suggestion, you can start by [Creating a CLI with Deno](/deno/build-a-cli-with-deno) written by my friend [Joni Oliveira]() and me.
+
+I hope you find it interesting.
+
+Best,
```
Comment (fsschmitt) [inline content/blog/deno/series-introduction.md:30]: Seems to be a bit lost "Best," not followed by a name, as it seems the short for `best regards`

File: content/blog/deno/series-introduction.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+We'll use these example applications to explore Deno and how it can be applied to common tasks. By the end of it, you will have a better understanding of this new JavaScript runtime and how it can be used.
+
+## Let's start!
+
+From now on it is up to you. 
+
+The above posts do not require to be read in any specific order, but if you want a suggestion, you can start by [Creating a CLI with Deno](/deno/build-a-cli-with-deno) written by my friend [Joni Oliveira]() and me.
+
+I hope you find it interesting.
```
Comment (fsschmitt) [inline content/blog/deno/series-introduction.md:28]:
```suggestion
I hope you find it interesting and have some fun while exploring these posts. If you have any questions, feel free to reach out to me on [Twitter](https://twitter.com/ampsantos0)!
```

**Pull request merged**